### PR TITLE
Upgraded oak to 1.8.3 + minor version bumps

### PR DESCRIPTION
**Pull request by ieugen**
* lots of upgrades for sling dependencies - minor version bumps

Signed-off-by: Ioan Eugen Stan <ieugen@apache.org>

### CODE REVIEW

File: src/main/provisioning/sling.txt [hunk 1]
```diff
@@ -58,31 +58,31 @@
...
     org.apache.sling/org.apache.sling.servlets.resolver/2.4.22
-    org.apache.sling/org.apache.sling.xss/2.0.4
+    org.apache.sling/org.apache.sling.xss/2.0.6
     javax.mail/mail/1.5.0-b01
     org.apache.geronimo.bundles/jstl/1.2_1
     org.apache.felix/org.apache.felix.http.whiteboard/4.0.0
 
 [artifacts startLevel=5]
-    org.apache.sling/org.apache.sling.starter.content/1.0.0
+    gr8pi.portal/gr8pi-portal-content/0.1.1
```
Comment (rombert) [inline src/main/provisioning/sling.txt]: I also assume you want to leave the starter content and not add the gr8pi stuff :-)

**Comment by ieugen** [comment]
Hi, done. See  #3. Thanks

**Comment by rombert** [comment]
You still have a changed `jackrabbit.version` in that PR.

**Comment by ieugen** [comment]
> You still have a changed jackrabbit.version in that PR. 
I'm not following you. Could you please recheck and be more specific?

**Comment by rombert** [comment]
At https://github.com/apache/sling-org-apache-sling-starter/pull/3/commits/b158af47da8ea2523fd298bdc05fcc32b5d2db62#diff-a8bdfa996f80a8529e1af2fa7e0c4868, you have changed the `jackrabbit.version` variable

**Comment by rombert** [comment]
We keep the Jackrabbit version in sync with the one that Oak uses. It can happen that Jackrabbit adds new APIs that need to be implemented by Oak so mismatches of Oak/Jackrabbit versions are problematic.

I pointed it out since it's in the PR that does not touch Jackrabbit/Oak.

**Comment by ieugen** [comment]
Now I get it. Updated both. Thanks. You are killing me :).

**Comment by rombert** [comment]
Thanks for your patience @ieugen . The change looks good to me. We need to add a Jira reference to the commit message ( probably SLING-7446 ) but I'll do that since I already pestered you a lot for this PR :-)

I asked for clarification on [SLING-7437](https://issues.apache.org/jira/browse/SLING-7437) since we may need changes there to support Oak 1.8. Once that is clear I'll merge the PR.

On an unrelated note, your 'From' and 'Signed-off-by' emails are not the same. Not a problem for me, but maybe you did not intend that...

**Comment by rombert** [comment]
@ieugen - I've tweaked this commit and pushed it to https://github.com/apache/sling-org-apache-sling-starter/tree/feature/oak-1.8 . We have some test failures after doing that, tracked at [SLING-7950](https://issues.apache.org/jira/browse/SLING-7950). Once those are out of the way, I'll merge the changes.

**Pull request closed without merging**